### DISCUSSION TITLE
Aspect Extraction supervised or unsupervised

### DISCUSSION THREAD

**Discussion by Tacacs-1101** [yangheng95/PyABSA#194] [General]
Are aspect extraction models supervised or unsupervised, and what method it used for aspect extraction. Can I use it on an open domain dataset?

**Comment by yangheng95** [comment]
It use the supervised method to extract aspects and classify sentiments after got the aspects. The method can be found at a paper:
https://www.sciencedirect.com/science/article/pii/S0925231220312534. You can annotate your dataset and fit the model on your data. Follow the instructions in ABSADatasets to annotate your APC dataset and convert it to ATEPC dataset then.
